style.css

```diff
@@ -0,0 +1,398 @@
+  display: flex;
+  gap: 20px;
+  padding: 2% 3%;
+}
+
+/* ============ End Hero Section ============== */
+
+/* ============ Works Section ============== */
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 10% 5%;
+}
+
+.works-title {
+  color: #1c1a19;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  margin: 3rem 0 4rem 0;
+}
+
+.works-title h2 {
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 400;
+}
+
+.underline {
+  width: 100%;
+  height: 1px;
+  background-color: #212121;
+}
+
+.projects {
+  display: grid;
+  gap: 2rem;
+}
+
+.project {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  color: #fff;
+}
+
+.project-text {
+  display: flex;
+  flex-direction: column;
+  gap: 0.6rem;
+}
+
+.other-projects h3,
+.other-projects p,
+.other-projects ul {
+  padding: 1% 5%;
+}
+
+.project .project-text h3 {
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 400;
+}
+
+.contact p {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: center;
+  padding: 1%;
+  margin: 2rem 0;
+}
+
+.about-me p {
+  color: #f4f5f7;
+  line-height: 24px;
+}
+
+.project .project-text p {
+  line-height: 24px;
+}
+
+.other-projects .project-image {
+  width: 100%;
+}
+
+.other-projects .project-text {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  z-index: 1;
+  padding-bottom: 0;
+}
+
+.first-project.project .project-text h3 {
+  color: #091e42;
+  margin-top: 1rem;
+}
+
+.first-project.project .project-text p {
+  color: #091e42;
+}
+
+.project-text .languages {
+  display: flex;
+  gap: 4%;
+}
+
+.languages li {
+  background-color: rgba(255, 255, 255, 0.24);
+  padding: 3% 4%;
+  font-size: 15px;
+  line-height: 20px;
+  font-weight: 500;
+}
+
+.first-project .languages li {
+  color: #091e42;
+  border: 1px solid #8993a4;
+}
+
+.about-me button {
+  align-self: flex-start;
+}
+
+.first-project .project-text button {
+  width: fit-content;
+}
+
+.project-text button:hover {
+  cursor: pointer;
+  color: #ff6b00;
+  background-color: #fff5e1;
+}
+
+/* ============ End Works Section ============== */
+
+/* ============ About Section ============== */
+
+.about {
+  background-color: #1c1a19;
+  margin-top: 3rem;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 5%;
+  background:
+    var(--hero-backgroung-color)
+    url('./images/about-me-illustration.svg') no-repeat right bottom;
+  min-height: 80vh;
+  padding-top: 30vw;
+}
+
+.about-me h2 {
+  font-size: 70px;
+  color: #ff6b00;
+  font-weight: 400;
+  letter-spacing: 0.37px;
+  line-height: 88px;
+}
+
+footer .underline {
+  background-color: #dfe1e6;
+}
+
+.about .underline {
+  background-color: #fff;
+  margin: 5px auto;
+  width: 90%;
+}
+
+.skills {
+  color: #fff;
+  font-size: 22px;
+  font-family: 'Inter', sans-serif;
+  display: flex;
+  flex-direction: column;
+  gap: 1.4rem;
+  align-items: center;
+  padding: 10rem 0;
```

Comment:
Awesome work on this. Just as a suggestion, remember to use relative dimensions when working with CSS to ensure your website remains responsive.